lookout/style/typos/correction_model.py

```diff
@@ -0,0 +1,408 @@
+import pickle
```

Comment:
This module would be better split into four modules (at least):
- modelforge.Model binding.
- feature extraction
- necessary training facilities (XGBoost)
- CLI to produce the model. The training process should be documented (how to call the CLI to get the required model on which data).